examples/audio_out.h

```diff
@@ -6,7 +6,9 @@
 ** file at : https://github.com/erikd/libsamplerate/blob/master/COPYING
 */
 
-typedef	struct AUDIO_OUT_s AUDIO_OUT ;
+typedef	struct AUDIO_OUT_s
+{	int magic ;
+} AUDIO_OUT ;
 
```

Comment:
With the definition of `AUDIO_OUT` removed from the file audio_out.c, this change should not be necessary.

Can you please try this and update the PR if it works?